metabase/metabase
# Close all descendant collections when clicking on an open parent

Solves this request https://github.com/metabase/metabase/pull/17680#issuecomment-909654305

### How to test

1. Have nested collections, at least 3 or 4 levels deep.

2. Go to a collections page

3. Click on the title of a collection that has children.

The collection should reveal its children.

4. Click on children until you can see several levels deep.

5. Click again on the title of the parent collection above.

The collection should hide its descendants.

6. Click again on the title of the parent collection above.

The collection should show only its direct children.

![Metabase Sidebar](https://user-images.githubusercontent.com/380816/131671767-d38de3a4-6dab-4caa-9445-aeb5c3668cf4.gif)
